apps/judicial-system/web/src/routes/Court/components/ReceptionAndAssignment/ReceptionAndAssignment.tsx

```diff
@@ -175,7 +175,11 @@ const ReceptionAndAssignment = () => {
       </FormContentContainer>
       <FormContentContainer isFooter>
         <FormFooter
-          previousUrl={`${constants.INDICTMENTS_COURT_OVERVIEW_ROUTE}/${id}`}
+          previousUrl={
```

Comment:
Found a bug. Going back from this screen in R-cases brought you to an indictment overview screen.